contracts/token/ERC6909/draft-ERC6909.sol

```diff
@@ -166,6 +165,9 @@ contract ERC6909 is Context, ERC165, IERC6909 {
      *
      * - `owner` cannot be the zero address.
      * - `spender` cannot be the zero address.
+     *
+     * Note: Setting `amount` to `type(uint256).max` is equivalent to setting an "infinite" allowance that won't
+     * be reduced when used.
```

Comment:
This will render it better in the docs page.

```suggestion
     * NOTE: Setting `amount` to `type(uint256).max` is equivalent to setting an "infinite" allowance that won't
     * be reduced when used.
```

Though, we may want to remove it as this is explicit in the ERC.

```suggestion
```

Reply:
Agree on removing in favor of documenting in the interface.